xgi/classes/hypergraph.py

```diff
@@ -935,6 +936,77 @@ def clear_edges(self):
         self._edge.clear()
         self._edge_attr.clear()
 
+    def merge_duplicate_edges(
+        self, rename="first", merge_rule="first", multiplicity=True
+    ):
+        """Merges edges which have the same members.
+
+        Parameters
+        ----------
+        rename : str, optional
+            Either "first", "tuple", or "new", by default "first"
+            If "first", the new edge ID is the first of the sorted
+            duplicate edge IDs. If "tuple", the new edge ID is a 
+            tuple of the sorted duplicate edge IDs. If "new", the
+            _edge_uid counter is selected and incremented.
+        merge_rule : str, optional
+            Either "first" or "union", by default "first"
+            If "first", takes the attributes of the first duplicate.
+            If "union", takes the set of attributes of all the duplicates.
+        multiplicity : bool, optional
+            Whether to return the multiplicity of the hyperedge,
+            by default True
+
+        Raises
+        ------
+        XGIError
+            If invalid rename or merge_rule specified.
+        
+        Warns
+        -----
+        If the user chooses merge_rule="union". Tells the
+        user that they can no longer draw based on this stat.
+        """
+        dups = []
+        hashes = defaultdict(list)
+        for idx, members in self._edge.items():
+            hashes[frozenset(members)].append(idx)
+
+        new_edges = list()
+        for members, dup_ids in hashes.items():
+            if len(dup_ids) > 1:
+                dups.extend(dup_ids)
+
+                if rename == "first":
+                    new_id = sorted(dup_ids)[0]
+                elif rename == "tuple":
+                    new_id = tuple(sorted(dup_ids))
+                elif rename == "new":
+                    new_id = next(self._edge_uid)
+                else:
+                    raise XGIError("Invalid ID renaming scheme!")
+
+                if merge_rule == "first":
+                    id = sorted(dup_ids)[0]
```

Comment:
```suggestion
                    id = min(dup_ids)
```

Reply:
dup_ids (edge IDs) are always integers right? Just checking that those will always work

Reply:
I checked and the min will work for both.